grammars/src/main/antlr/adl_rules14.g4

```diff
@@ -18,7 +18,7 @@ assertion_list: (assertion (';')?)+ ;// {_input.LA(1) == WS || _input.LA(1) == L
 
 assertion: variableDeclaration | booleanAssertion;
 
-variableDeclaration: SYM_VARIABLE_START identifier SYM_COLON identifier SYM_ASSIGNMENT expression;
```

Comment:
`adl_rules.g4` has
```g4
variableDeclaration: VARIABLE_DECLARATION SYM_ASSIGNMENT expression;
```

Which one is correct?

Reply:
adl_rules.g4 is correct. It's very minor since as far as I know there is no adl 1.4 archetype with rules. I've changed it in the 14 rules grammar.